### PR TITLE
add switch docs

### PR DESCRIPTION
**Pull request by e111077**
Related #348

### CODE REVIEW

File: packages/switch/README.md [hunk 1]
```diff
@@ -1,64 +1,92 @@
...
+| `checked`  | `boolean` | `false` | Whether or not the switch should be checked / activated.
+| `disabled` | `boolean` | `false` | Disables the input and sets the disabled styles.
 
-      ```npm i -g polymer-cli```
+### Methods
+*None*
 
-  1. Run the development server and open a browser pointing to its URL:
+### Events
+*None*
```
Comment (aomarks) [inline packages/switch/README.md]: Add `change` event docs.

Comment (e111077) [reply packages/switch/README.md]: stolen from checkbox

File: packages/switch/README.md [hunk 2]
```diff
@@ -74,10 +75,14 @@ npm install @material/mwc-switch
...
 ### Methods
+
 *None*
 
 ### Events
-*None*
+
+| Event Name | Target         | Detail | Description
+| ---------- | -------------- | ------ | -----------
+| `change`   | `mwc-checkbox` | `{}`   | Fired when the user modifies the swtich `checked` state from an input device interaction. Note that, like [native `<input>`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event), the `change` event is *not* fired when the `checked` property is set from JavaScript.
```
Comment (aomarks) [inline packages/switch/README.md]: s/swtich/switch

Comment (aomarks) [reply packages/switch/README.md]: also mwc-checkbox/mwc-switch

**Pull request merged**